pytest/tests/loadtest/locust/common/base.py

```diff
@@ -14,19 +14,24 @@
 import logging
 import multiprocessing
 import pathlib
+import requests
 import sys
 import time
 
 sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
 
-DEFAULT_TRANSACTION_TTL_SECONDS = 20
+DEFAULT_TRANSACTION_TTL_MILLISECONDS = 1_800_000
```

Comment:
A few ideas on how to make this easier to parse for the reader:
- Add a comment with an easy-to-parse timeout:
```python
DEFAULT_TRANSACTION_TTL_MILLISECONDS = 1_800_000  # 30 minutes.
```
- Use a structured type like `datetime.timedelta` [1] to represent this TTL:
```python
DEFAULT_TRANSACTION_TTL = datetime.timedelta(minutes=30)
```

[1] https://docs.python.org/3/library/datetime.html#timedelta-objects

Reply:
To pass the argument to retrying, I'd have to do something like this, wouldn't I?

```python
    @retry(wait_fixed=500,
           stop_max_delay=total_milliseconds = DEFAULT_TRANSACTION_TTL_MILLISECONDS.seconds * 1000 + DEFAULT_TRANSACTION_TTL_MILLISECONDS.microseconds / 1000,
           retry_on_exception=is_tx_unknown_error)
```

unless there is a simpler way, I think I prefer the comment

Reply:
I think `DEFAULT_TRANSACTION_TTL.total_seconds() * 1000` should do the job.

https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds

Reply:
Ah, didn't see that but yes that's nicer!